py/torch_tensorrt/dynamo/_exporter.py

```diff
@@ -276,6 +278,7 @@ def inline_trt_modules(
                 (trt_module_node.args, trt_module.engine),
             )
             trt_node.meta["val"] = []
+            assert num_outputs
```

Comment:
Can you make this assert more clear? like `assert num_outputs > 0` or something?

Reply:
done